evap/staff/tests.py

```diff
@@ -1229,20 +1228,14 @@ def test_user_with_same_email(self):
         data = {"username": "uiae", "email": user.email}
         form = UserForm(data=data)
         self.assertFalse(form.is_valid())
-        form = ContributorUserForm(data=data)
-        self.assertFalse(form.is_valid())
 
         data = {"username": "uiae", "email": user.email.upper()}
         form = UserForm(data=data)
         self.assertFalse(form.is_valid())
-        form = ContributorUserForm(data=data)
-        self.assertFalse(form.is_valid())
 
         data = {"username": "uiae", "email": user.email.upper()}
         form = UserForm(instance=user, data=data)
         self.assertTrue(form.is_valid())
-        form = ContributorUserForm(instance=user, data=data)
-        self.assertTrue(form.is_valid())
```

Comment:
i know, but still, you removed some code and replaced it with new code, and removed the tests without replacing them with new ones. what i want to say is that a test testing the form would be cool.